src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1551,6 +1572,8 @@ def add_waveforms(
                 sorting_copy.set_property(prop, tm[prop])
 
     add_electrodes_info(recording, nwbfile=nwbfile, metadata=metadata)
+    electrode_group_indices = get_electrode_group_indices(recording, nwbfile=nwbfile)
+    unit_electrode_indices = [electrode_group_indices] * len(sorting.unit_ids)
```

Comment:
This will also allow to automatically handle sparsity, when that is propagated to the `WaveformExtractor` object in SpikeInterface (see [PR](https://github.com/SpikeInterface/spikeinterface/pull/1152))